src/test/java/edu/ucsb/cs56/pconrad/SparkDemo01Test.java

```diff
@@ -0,0 +1,38 @@
+package edu.ucsb.cs56.pconrad;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+import org.junit.Before;
+import org.junit.After;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+public class SparkDemo01Test {
+
+
+	private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+	private final ByteArrayOutputStream errContent = new ByteArrayOutputStream();
+	private final PrintStream originalOut = System.out;
+	private final PrintStream originalErr = System.err;
+	
+	@Before
+	public void setUpStreams() {
+		System.setOut(new PrintStream(outContent));
+		System.setErr(new PrintStream(errContent));
+	}
+
+	@After
+	public void restoreStreams() {
+		System.setOut(originalOut);
+		System.setErr(originalErr);
+	}
+
+	@Test
+	public void test_out() {
+		assertTrue(true);
+	}
+	
+}
```

Comment:
You still need to properly unit test your code before pushing into master. Send the PR once you've gone ahead and fully tested and double checked all your code.

Reply:
Yeah, ik. Conrad was showing me how to branch 😅 and then send the branch to our main App. But we just changed one thing, and realize that after I sent the request. Sorry.